.travis.yml

```diff
@@ -4,9 +4,18 @@ php:
   - 5.3
   - 5.4
   - 5.5
+  - 5.6
+  - 7.0
+  - hhvm
 
-before_script:
-  - curl -s http://getcomposer.org/installer | php
-  - php composer.phar install --dev
+## Cache composer
+sudo: false
 
-script: phpunit
\ No newline at end of file
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+install:
+  - travis_retry composer install --prefer-dist --no-interaction
+
+script: vendor/bin/phpspec run
```

Comment:
I updated the PR description.
This was probably just a boilerplate skeleton setup with the default travis file. It doesn't seem to be activated, see https://travis-ci.org/antonioribeiro/google2fa
So after this, the hooks should be activated by the maintainer.